cli/internal/taskhash/taskhash.go

```diff
@@ -332,7 +335,9 @@ func (th *Tracker) CalculateTaskHash(packageTask *nodes.PackageTask, dependencyS
 }
 
 // GetExpandedInputs gets the expanded set of inputs for a given PackageTask
-// Thes was stored during CalculateFilesHash, so that method must run first
+// Thes was stored during CalculateFilesHash(), so that method must run first.
+// Reads from the underlying map in this method do NOT need to be protected by a mutex
```

Comment:
I think this comment would be better off attached to the map declaration itself. What you're describing is a property of the map, not this method. Another method that accessed the same map would need the same comment.

IIUC, this map is calculated once and then considered immutable after `CalculateFileHashes` returns.